wondrous-app/utils/hooks.ts

```diff
@@ -308,3 +310,33 @@ export const useSteps = (defaultStep = 0) => {
   const [step, setStep] = useState(defaultStep);
   return { step, setStep };
 };
+
+export const usePermissions = (entity, isTaskProposal = false) => {
```

Comment:
nice! cool eventually we'll use this across other things, rn let's just keep it as is